google/resource_storage_bucket_imapolicy_test.go

```diff
@@ -0,0 +1,182 @@
+package google
+
+import (
+	"bytes"
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	storagev1 "google.golang.org/api/storage/v1"
+)
+
+func testBucketIamName() string {
+	return fmt.Sprintf("%s-%d", "tf-test-iam-bucket", acctest.RandInt())
```

Comment:
You can just use `acctest.RandomWithPrefix("tf-test-iam-bucket")` which is a little less convoluted.